bcda/bcdacli/cli.go

```diff
@@ -684,7 +684,7 @@ func setBlacklistState(cmsID string, blacklistState bool) error {
 }
 
 // CCLF file name pattern and regex
-const cclfPattern = `((?:T|P)\.BCD\..*\.ZC\d*)Y(\d{2}\.D\d{6}\.T\d{7})`
+const cclfPattern = `((?:T|P).*\.ZC[A-B0-9]*)Y(\d{2}\.D\d{6}\.T\d{7})`
```

Comment:
Im assuming the check for `[A-B...` will always directly follow `ZC` maybe something like this will be more specific.
`((?:T|P).*\.ZC[A-B]?[0-9]*)Y(\d{2}\.D\d{6}\.T\d{7})`

Reply:
Since we can have digits OR A/B, we'll leave the regex as is